test.js

```diff
@@ -1,6 +1,41 @@
+const os = require("os")
 const test = require("ava")
 const minecraftPath = require(".")
 
-test("main", t => {
-	t.is(typeof minecraftPath(), "string")
+test("type of minecraftPath return", t => {
+	const originalPlatform = process.platform
+
+	const platforms = ["win32", "darwin", "linux", "android"]
+
+	t.plan(platforms.length)
+
+	for (const platform of platforms) {
+		Object.defineProperty(process, "platform", {
+			value: platform
+		})
+
+		t.is(typeof minecraftPath(), "string")
+	}
+
+	Object.defineProperty(process, "platform", {
+		value: originalPlatform
+	})
+})
+
+test("platform not supported error", t => {
+	const supported = ["win32", "darwin", "linux", "android"]
+
+	if (supported.includes(process.platform)) {
+		t.notThrows(() => minecraftPath(), `${process.platform} is not supported!`)
+	} else {
+		t.throws(() => minecraftPath(), `${process.platform} is not supported!`)
```

Comment:
The second argument can't be a string. Instead, it is an object: `{ message: string }`

https://github.com/avajs/ava/blob/master/docs/03-assertions.md#throwsfn-expectation-message

Reply:
In `.notThrows(fn, message?)` there's not the `expectation?` like in `.throws(fn, expectation?, message?)`.

So it should be:
```javascript
if (supported.includes(process.platform)) {
	t.notThrows(() => minecraftPath(), `${process.platform} is not supported!`)
} else {
	t.throws(() => minecraftPath(), { message: `${process.platform} is not supported!` })
}
```

Reply:
Yes